internal/ingress/annotations/alias/main_test.go

```diff
@@ -76,4 +73,6 @@ func TestParse(t *testing.T) {
 			t.Errorf("expected %v but returned %v, annotations: %s", testCase.expected, result, testCase.annotations)
 		}
 	}
+
+	parser.EnableAnnotationValidation = true
```

Comment:
I think if some test fail we will never reach this point, and have some undesired behaviors on other tests.

Maybe replace the defer func to a t.Cleanup call